examples/cp4s/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    ibm = {
+      source  = "ibm-cloud/ibm"
+      version = "1.34"
```

Comment:
Do you have any terraform provider version dependency ?

If we dont mention the version it takes latest version always

Reply:
Yes, we found that this version works.  Later versions introduced a bug.